src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-23-expected.json

```diff
@@ -151,27 +199,24 @@
       }
     },
     {
-      "fullUrl": "urn:uuid:b5aae309-4caf-c2cd-b087-0fd6db6dd058",
+      "fullUrl": "urn:uuid:",
       "resource": {
-        "resourceType": "Provenance",
-        "id": "b5aae309-4caf-c2cd-b087-0fd6db6dd058",
-        "agent": [
-          {
-            "type": {
-              "coding": [
-                {
-                  "code": "author",
-                  "system": "http://terminology.hl7.org/CodeSystem/provenance-participant-type"
-                }
-              ]
-            }
-          }
-        ],
-        "recorded": "2005-01-10T04:55:04Z"
+        "resourceType": "Practitioner"
       },
       "request": {
         "method": "PUT",
-        "url": "Provenance/b5aae309-4caf-c2cd-b087-0fd6db6dd058"
+        "url": "Practitioner/"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:b065b233-8738-f1ec-1314-d61e4aac2c9e",
+      "resource": {
+        "resourceType": "Practitioner",
+        "id": "b065b233-8738-f1ec-1314-d61e4aac2c9e"
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Practitioner/b065b233-8738-f1ec-1314-d61e4aac2c9e"
       }
     },
```

Comment:
There seem no useful information in resources like this. Are we expecting to generate them in the output?

Reply:
I'm also not sure about how to handle this "empty" resource.
@ranvijaykumar Do you have suggestions?